openfe/utils/visualization.py

```diff
@@ -83,6 +83,28 @@ def _get_unique_bonds_and_atoms(mapping: Dict[int, int],
     return uniques
 
 
+def _draw_molecules(d2d, mols, atoms_list, bonds_list, atom_colors,
```

Comment:
do we need / want types here?

Reply:
types here are pretty ugly... which is, I guess, why I should add them. Done now.